website/content/en/v0.26.1/upgrade-guide.md

```diff
@@ -41,6 +41,18 @@ Karpenter ships with a few Custom Resource Definitions (CRDs). These CRDs are pu
     ```
     helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version vx.y.z --namespace karpenter --create-namespace
     ```
+  * Upgrading from older karpenter version that did not include `awsnodetemplates.karpenter.k8s.aws` labels and annotations requires manual CRDs annotations before issuing `helm upgrade`. 
+
+In the case of `invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"` execute:
```

Comment:
Can you put these details under the troubleshooting in the "Install" section and then link to this troubleshooting in the upgrade guide?

Reply:
moved the changes to troubleshooting under Installation and referenced it in upgrade-guide